etc/landing/index.html

```diff
@@ -5,17 +5,17 @@
   </head>
   <body>
     <h1>Vostok Spaceport</h1>
-    <p>Architecture and brief description &ndash; <a href="http://vostok.tools">http://vostok.tools</a></p>
     <h2>Web services</h2>
     <table id="services">
+      <!-- not starting now
```

Comment:
Why not?

Reply:
Because now it is impossible to run these components into hercules. We are waiting https://yt.skbkontur.ru/issue/HERCULES-582.